aptos-move/framework/move-stdlib/sources/cmp.move

```diff
@@ -0,0 +1,98 @@
+module std::cmp {
+    const EQUAL: u8 = 1;
+    const LESS_THAN: u8 = 0;
+    const GREATER_THAN: u8 = 2;
+
+    enum Ordering has copy, drop {
+        /// First value is less than the second value.
+        LessThan,
```

Comment:
I'd strongly prefer to follow rust here, and have these be `Less`, `Greater` and `Equal`